release-images.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Not needed for now, but I love this work for automated release process, save this -- keep it handy -- we will do something similar in `canfar-library`